untitled/src/main/java/com/aca/homework/week15/sql/company/sql-company.txt

```diff
@@ -0,0 +1,18 @@
+There are a lot of companies and users that can not exist outside of the company.
```

Comment:
because the user can not exist outside of the company I suggest creating company and company_user tables only.